### ISSUE TITLE
Migrate .Net Framework Reliable Services to SF Mesh

### ISSUE THREAD

**Issue by PrabhujiMudragalla** [Azure/service-fabric-mesh-preview#250]
Hi Team, 

We are eager to migrate our application to SF Mesh. I am afraid about about the limitations of SF Mesh. 

My application is build up on .Net Framework Service Fabric Reliable services which include following capabilities:
1. Stateful Services with Reliable collections
2. Reliable actors
3. Backup and Restore mechansim
4. Event Grid Messaging 

And many more..

All this components are implemented using .Net Framework. So can you provide me a mechansim to migrate all this components without affecting the business. 

It would be greatful if you provide the roadmap of SF Mesh. Such that we can plan our deliverables. 

Thanks Prabhuji

**Comment by tomkerkhove** [comment]
Reliable collections & actors are not there (yet) so I think you'll have to wait a while but you can already start running your applications as Windows containers

**Comment by ai2admin** [comment]
Is there a step-by-step guide for creating a yaml/json for SF Mesh that uses my containers? I tried to modify then one produced by Visual Studio for .NET Core containers and I failed.

I already have my .NET Framework containers deployed to Azure registry and used by a regular SF cluster. It works. Now I need to cover the gap with giving those a try. I imagine I'm not the only one who's looking for a step-by-step guide to deliver NET Framework containers to SF Mesh.

**Comment by joergjo** [comment]
For the time being, I typically pick one of the (ARM) [templates](https://github.com/Azure-Samples/service-fabric-mesh/tree/master/templates) provided in the samples repo and work from there.

**Comment by ai2admin** [comment]
This extension requires Visual Studio Code. Is there a version that works with Visual Studio Professional?

**Comment by MikkelHegn** [comment]
The Service Fabric Mesh SDK preview and VS extension for Mesh, which eventually will also support debugging .net apps in containers. But if all you need is the arm template to deploy, the VSCode extension will give you that.

**Comment by ai2admin** [comment]
@MikkelHegn is there a manual on how to use it? E.g. I type "Service Fabric Mesh: Create Application" and hit [Enter] in the command and it gives me a couple of errors saying "Please open a workspace directory first" and "Path must be a string. Received undefined", but it's cryptic. How do I know what, where and how to set up as the command input?

**Comment by MikkelHegn** [comment]
Please check this repo: https://github.com/michaelfery/vscode-sf-mesh-tools @michaelfery (the author) can help.

**Comment by ai2admin** [comment]
@MikkelHegn,  yes, I saw the repo. How do I diagnose the error that I'm given?

**Comment by MikkelHegn** [comment]
There is one issue opened over there in regards to create application: https://github.com/michaelfery/vscode-sf-mesh-tools/issues/7 If that's not what you see, please open an issue in that repo.

I do remember that you need to have a folder open for that to work. You can try to fall back to the yeoman generator, which the extension relies on: https://github.com/michaelfery/generator-azuresfmesh

**Comment by michaelfery** [comment]
A PR for adding an explicit error when no workspace is opened has been merged in michaelfery/vscode-sf-mesh-tools#10
